exporters/trace/README.md

```diff
@@ -5,44 +5,88 @@
 
-## Installation
+### Installation
   Not currently ready as a package to be imported. [Issue](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/issues/6) is currently being dealt with.  
-  For now, one would have to clone this GitHub repo. To do so, run in the command line:
-  ```git
-  git clone https://github.com/GoogleCloudPlatform/opentelemetry-operations-java
+  For now, one would have to clone this GitHub repo and import it as a separate project (see example). To do so, run in the command line:
+  ```sh
+  git clone git@github.com:GoogleCloudPlatform/opentelemetry-operations-java.git
   ```
-## Usage
+
+### Usage
   If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in Authentication.  
-    
-  The TraceExporter constructor takes in three parameters: `projectId: String` for your GCP project ID, `traceServiceClient: TraceServiceClient` for the trace service client to eventually batch write spans, and `fixedAttributes: Map<String, AttributeValue>` for the fixed attributes of a span.  
-  So, we need to import the following: 
-  ```java
-  import com.google.cloud.trace.v2.TraceServiceClient;
-  import com.google.devtools.cloudtrace.v2.AttributeValue;
-  import com.google.devtools.cloudtrace.v2.ProjectName;
-  import com.google.devtools.cloudtrace.v2.Span;
-  ```
-  Declare and initialize the variables that will be used in the constructor parameters.  
-  Then, we can create and register TraceExporter, for example:
-  ```java
-  TraceExporter javaTraceExporter = new TraceExporter(projectId, traceServiceClient, fixedAttributes);
-  OpenTelemetrySdk.getTracerProvider().addSpanProcessor(SimpleSpanProcessor.newBuilder(this.javaTraceExporter).build());
-  ```
-  Start tracing and collecting SpanData.  
-  Spans can be created by importing and using global `opentelemetry-java` API packages, for example:  
+
+#### Create the exporter
+
+This uses the default configuration for authentication and project ID.
+
+```java
+public class MyMainClass {
+  public static void main(String[] args) throws Exception {
+    this.traceExporter = TraceExporter.createWithDefaultConfiguration();
+    // ...
+  }
+}
+```
+
+Or you can customize it with a TraceConfiguration object
+```java
+public class MyMainClass {
+  public static void main(String[] args) throws Exception {
+    this.traceExporter = TraceExporter.createWithConfiguration(
+      TraceConfiguration.builder().setProjectId("myCoolGcpProject").build()
+    );
+    // ...
+  }
+}
+```
+
+
+  Then, we can connect TraceExporter to OpenTelemetry, for example:
   ```java
-  String operationName = "receive"; //or whatever other operation
-  Span span = this.tracer.spanBuilder(operationName).startSpan();
+  OpenTelemetrySdk.getTracerProvider().addSpanProcessor(SimpleSpanProcessor.newBuilder(this.traceExporter).build());
```

Comment:
```suggestion
  OpenTelemetrySdk.getTracerProvider().addSpanProcessor(SimpleSpanProcessor.newBuilder(exporter).build());
```

Reply:
It probably makes sense to just move this code snippet into the first example with `createWithDefaultConfiguration` and rephrase the comment to "create exporter and register it in the OpenTelemetry SDK"